### DISCUSSION TITLE
On the Helper Scripts Website the Zabbix Default Login Credential is writen admin instead of Admin

### DISCUSSION THREAD

**Discussion by mogultekin** [community-scripts/ProxmoxVE#71] [General]
On the Helper Scripts Website the Zabbix Default Login Credential is writen admin instead of Admin. When tried to login for the first time looking at the written name. Then I searched to reset it. There I saw and remembered the default user is Admin. It will be good to correct it. 

BR,
mogultekin

**Comment by MickLesk** [comment]
@BramSuurdje 

https://github.com/community-scripts/ProxmoxVE/issues/75

**Comment by bramsuurdje** [comment]
fixed in issue #75. thank you.
